Analysis/HiggsTauTau/src/HTTCategories.cc

```diff
@@ -17,7 +17,8 @@ namespace ic {
   HTTCategories::HTTCategories(std::string const& name) : ModuleBase(name), 
       channel_(channel::et), 
       era_(era::data_2012_rereco),
-      strategy_(strategy::paper2013) {
+      strategy_(strategy::paper2013),
+      mc_(mc::summer12_53X){
```

Comment:
To my knowledge I didn't add this so I'm not sure why it's there. I've removed anyway